plugins/ScheduledReports/templates/index.twig

```diff
@@ -32,6 +32,7 @@
 </div>
 
 <script type="text/javascript">
+    var timeZoneDifference = {{ timeZoneDifference }} / 3600;
```

Comment:
Small thing, but I would move converting this value to adjustHourToTimezone() function or to the controller. Currently  timeZoneDifference is created and modified in three places: in controller, here in template and in JavaScript. It may cause some confusion in future.